pkg/artifactory/resource_artifactory_user_test.go

```diff
@@ -78,6 +78,10 @@ func testAccCheckUserDestroy(id string) func(*terraform.State) error {
 
 		user, resp, err := client.V1.Security.GetUser(context.Background(), rs.Primary.ID)
 
+		if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.